pom.xml

```diff
@@ -2,14 +2,14 @@
 
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <groupId>com.xebialabs.jira.addons</groupId>
+    <groupId>org.venkatk.jira.addons</groupId>
```

Comment:
Hi

Actually I had modified the code further after I had issued a pull request, to be able to upload the plugin to Atlassian Marketplace. I was suggested  by @vanrobstone to go ahead and upload the plugin and hence had to change the groupId to something other than com.xebialabs since I was told Xebia had lot more to enhance on the plugin before Xebia can upload / support it from Atlassian Market place..